src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.navigation__wraper {
```

Comment:
Good, but follow the sequence of elements. Wrapper should be higher than nav list